addons/calendar/tests/test_event_recurrence.py

```diff
@@ -741,21 +741,45 @@ def test_unlink_recurrence_future(self):
 
     def test_unlink_recurrence_wizard_next(self):
         event = self.events[1]
-        wizard = self.env['calendar.popover.delete.wizard'].create({'record': event.id})
+        wizard = self.env['calendar.popover.delete.wizard'].with_context(
+            form_view_ref='calendar.calendar_popover_delete_view').create(
+                {
+                    'record': event.id
+                }
+        )
         form = Form(wizard)
         form.delete = 'next'
         form.save()
         wizard.close()
+        wizard_delete = self.env['calendar.popover.delete.wizard'].with_context(
+            form_view_ref='calendar.view_event_delete_wizard_form',
+            default_recurrence='next'
+        ).create({'record': event.id})
+        form_delete = Form(wizard_delete)
+        form_delete.save()
+        wizard_delete.action_send_mail_and_delete()
         self.assertTrue(self.recurrence)
         self.assertEqual(self.events.exists(), self.events[0])
 
     def test_unlink_recurrence_wizard_all(self):
         event = self.events[1]
```

Comment:
This is a good opportunity to improve this test's readability, could you please apply the save suggestion as above? (adding the comments through the test execution.)